Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/ModelData.xml

```diff
@@ -1021,8 +1021,26 @@
     </CModel>
     <CModel id="AP_MarauderMercenaryPortrait" parent="FXAPortrait" Race="Terran">
         <Model value="Assets\Portraits\Terran\MarauderMercenaryPortrait\MarauderMercenaryPortrait.m3"/>
+        <Image value="Assets\Textures\maruader_merc_collection_static.dds"/>

```

Comment:
Verified this typo "maruader" is correct and matches what's in the gamefiles. Blizzard typo `:P`